### PR TITLE
update homepage to sync with github repo

### PR DESCRIPTION
**Pull request by adam3q**
1. Add Quickstart and FAQ button.
2. Improve the description of the Design section.
3. Replace the link to the Dragonfly section with the Helm doc.
4. Replace outdated remote graph driver with Docker/Moby docs.

### CODE REVIEW

File: components/Design.vue [hunk 1]
```diff
@@ -47,11 +47,11 @@
...
           <div class="intro">
-            <div class="title">Extensible and Flexible</div>
+            <div class="title">Flexible</div>
             <div class="desc">
-              Confidential container, vulnerability scan, support both runC
-              native container and
-              <span class="inline-highlight">Kata Container</span>.
+              Supports container runtimes such as runC and 
+              <span class="inline-highlight">Kata</span>,
+              and provides Confidential Containers and vulnerability scanning capabilities
```
Comment (imeoer) [inline components/Design.vue]: `provides Confidential Containers` The statement seems a little strange.

File: components/Design.vue [hunk 2]
```diff
@@ -102,10 +102,10 @@
             <img src="~/static/images/helmet.png" alt="" class="emoji" />
           </div>
           <div class="intro">
-            <div class="title">End-to-end Data Integrity</div>
+            <div class="title">POSIX Compatibility</div>
             <div class="desc">
-              File metadata and blob data can be both validated in transmission
-              and at runtime.
+              In-Kernel EROFS or FUSE filesystems together 
+              with overlayfs provide full POSIX compatibility.
```
Comment (imeoer) [inline components/Design.vue]: How about `In-kernel EROFS, FUSE, and VirtioFS support provides full POSIX compatibility.`

File: components/Design.vue [hunk 3]
```diff
@@ -116,8 +116,8 @@
           <div class="intro">
             <div class="title">Compatible with Ecosystem</div>
             <div class="desc">
-              Support with Registry, OSS, NAS, Shared Disk, and P2P service,
-              compatible with the OCI spec.
+              Support with Registry, OSS, NAS, Shared Disk, and P2P service.
+              Compatible with the OCI images, and provide native eStargz images support.
```
Comment (imeoer) [inline components/Design.vue]: We can't promise the estargz support now.

**Review by imeoer — APPROVED** [review]

File: components/Design.vue (hunk 2 above)
Comment (adam3q) [reply components/Design.vue]: Great.

File: components/Design.vue (hunk 3 above)
Comment (adam3q) [reply components/Design.vue]: OK, I removed it.

File: components/Design.vue (hunk 1 above)
Comment (adam3q) [reply components/Design.vue]: I changed it to `Supports container runtimes such as runC, Kata and confidential containers`

**Pull request merged**